include/freerdp/autodetect.h

```diff
@@ -28,6 +28,16 @@
 extern "C"
 {
 #endif
+
+	typedef enum
+	{
+		AUTODETECT_STATE_INITIAL,
+		AUTODETECT_STATE_REQUEST,
+		AUTODETECT_STATE_RESPONSE,
+		AUTODETECT_STATE_COMPLETE,
+		AUTODETECT_STATE_FAIL
```

Comment:
Nit: Could also end with a `,` here. Then, in case of state additions, we won't have change unrelated existing lines.

Reply:
that would produce quite a few warnings during build as gcc/clang complain about that when set to pedantic/everything ;)